meilisearch/errors.py

```diff
@@ -1,7 +1,10 @@
+from __future__ import annotations
```

Comment:
Technically it isn't needed but it doesn't hurt anything to have it. I added it because I have had cases where I didn't put it, then later went back and added something that need it and forgot to import `__future__`. The issue there is python 3.10+ works without it so everything seems good, but then errors show up if you try to use < 3.10. 

So it's just there as a future `__future__` :smile: safety net. If you would rather remove I can.